dashboard/settings.py

```diff
@@ -307,6 +307,9 @@
 # This is fixed from UDW
 UDW_ID_PREFIX = config("UDW_ID_PREFIX", default="17700000000")
 
+# This is fixed from UDW
+UDW_FILE_ID_PREFIX = config("UDW_FILE_ID_PREFIX", default="1770000000")
```

Comment:
I wonder if this is ever going to ever be different than UDW_ID_PREFIX?

Reply:
This value is also not in the .env.sample.

Reply:
I think for this we should just set the default value of UDW_ID_PREFIX and just add it to the sample, if we think it *might* be different.

Reply:
The UDW_FILE_ID_PREFIX is less one zero compared to UDW_ID_PREFIX. Don't ask me how I found  out  -:(

Reply:
I will add that into .env.sample

Reply:
.env.sample updated